docs/architecture/adr/adr-006-in-cluster-db.md

```diff
@@ -0,0 +1,170 @@
+# ADR 006: In-cluster Database
+
+## Context
+
+We have known, since summer 2024, that Microsoft had decided to sunset Azure
+database for MariaDB. Their recommendation has been to migrate to their Flexible
+MySql database which they have provided pretty well documented. Back then we had
```

Comment:
"which they have provided pretty well documented" - I don't know what this means, but it can also just be removed. The rest of the sentence is good and fine.